### ISSUE TITLE
[FR] More Configurable Options

### ISSUE THREAD

**Issue by Lordfirespeed** [LethalCompanyModding/Enhancer#4]
It would be great if the mod could support configuring some additional options!

### Quota Options

The following variables are part of `QuotaSettings` and require no implementation other than updating their values:
- `baseIncrease`
- `increaseSteepness`
- `randomizerMultiplier`
- `startingCredits`
- `startingQuota`

#### Maximum Death Penalty %
The % of your money that you will lose if all players die and are unrecovered.

#### Death Penalty Harshness Scaling

For ease of reading, allow these variables:
- Let $C$ be the amount of cash the lobby has
- Let $M$ be the maximum death penalty %
- Let $P$ be the number of players in the lobby
- Let $P_D$ be the number of players that died without their bodies being recovered
Let $P_R$ be the number of players that died whose bodies were recovered

##### Vanilla Behaviour

In Vanilla, players lose a linear % of their cash per unrecovered player, and a lower linear % of their cash per recovered player. 

That is, cash loss = $C\times (P_D + 0.4\times P_R)\times M/P$

##### Proposed Behaviour

Add a configured floating-point value, say $\lambda$.

The value would allow configuring the penalty 'harshness' so that fewer unrecovered deaths -> less penalty.

Cash loss = $C\times (P_D + 0.4\times P_R)^\lambda \times M/P^\lambda$

With the default value of $1$, vanilla functionality is preserved.

Setting $\lambda=2$, in a lobby of 4 players, with maximum penality = 80%:
- Loss with 1 death = 5%
- Loss with 2 deaths = 20%
- Loss with 3 deaths = 45%
- Loss with 4 deaths = 80%
